infrastructure/main.tf

```diff
@@ -62,7 +62,9 @@ locals {
   subnetwork_name = var.create_network ? module.custom-network[0].subnets_names[0] : var.subnetwork_name
   region          = length(split("-", var.cluster_location)) == 2 ? var.cluster_location : ""
   regional        = local.region != "" ? true : false
```

Comment:
In this case regional should be false?

Reply:
this is so complicated.
I think region = var.cluster_location if it's a regional cluster.
so local.region != "" is true for regional clusters. so this is correct.

@umeshkumhar this naming is super confusing

Reply:
Just to clarify.. we take **cluster_location** input only.

Then we check if user provided region or zone by splitting on hypen. Once done, we check if region is set that means cluster will be regional, so make it true, else false.